app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -524,10 +533,19 @@ class DefaultToolbarMenu(
         menuItems
     }
 
+    private fun bookmarkListener() {
```

Comment:
Maybe some variation of our handleX naming.

```suggestion
    private fun handleBookmarkItemTapped() {
```